numba/tests/test_array_methods.py

```diff
@@ -630,13 +642,32 @@ def test_sum(self):
         self.assertPreciseEqual(pyfunc(a), cfunc(a))
         # OK
         self.assertPreciseEqual(pyfunc(a, 0), cfunc(a, 0))
-        
+
+    def test_sum_kws(self):
+        pyfunc = array_sum_kws
+        cfunc = jit(nopython=True)(pyfunc)
+        # OK
+        a = np.ones((7, 6, 5, 4, 3))
+        self.assertPreciseEqual(pyfunc(a, axis=1), cfunc(a, axis=1))
+        # OK
+        self.assertPreciseEqual(pyfunc(a, axis=2), cfunc(a, axis=2))
+
+    def test_sum_const(self):
+        pyfunc = array_sum_const_multi
+        cfunc = jit(nopython=True)(pyfunc)
+
+        arr = np.ones((3, 4, 5, 6, 7, 8))
```

Comment:
Glad you mention this.  Cause I forgotten to test this and a small addition will enable the support of negative axis.